.gitignore

```diff
@@ -37,10 +37,6 @@ cover.html
 # Project-local glide cache, RE: https://github.com/Masterminds/glide/issues/736
 .glide/
 
-# Kubernetes Generated files - skip generated files, except for vendored files
-zz_generated.*
-!vendor/**/zz_generated.*
```

Comment:
keep vendor part?

Reply:
If skipping `zz_generated.*` files has already been removed, maybe there is no need to exclude files in the vendor directory.